sdk/monitor/monitor-query/README.md

```diff
@@ -0,0 +1,174 @@
+# Azure Monitor Workspace query client library for JavaScript
+
+[Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/overview) is a comprehensive solution for collecting, analyzing, and acting on telemetry from your cloud and on-premises environments.
+
+Use the client library for Azure Monitor to:
+
+- Query logs using the [Kusto query language][kusto_query_language]
+- Query metrics
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/monitor/monitor-query/) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/monitor-query) |
+[API reference documentation]<!--(https://docs.microsoft.com/javascript/api/@azure/monitor-query)--> |
+[Product documentation][azure_monitor_product_documentation]
+[Samples][samples]
+
+## Getting started
+
+### Install the package
+
+```bash
+npm install @azure/monitor-query
+```
+
+### Prerequisites
+
+- You must have an [Azure Subscription](https://azure.microsoft.com) and an [Azure Monitor][azure_monitor_product_documentation] resource to use this package.
+- Node.js version 10.x.x or higher
```

Comment:
Nice.

Reply:
It did feel pretty good to write that. :)